src/components/Banner.js

```diff
@@ -3,19 +3,24 @@ import '../styles/banner.scss'
 import { NavLink } from 'react-router-dom'
 
 export default function Banner() {
-    return (
-        <div className="banner-container">
-            <div className="banner">
-                <div className="tag-line">
-                    <h2>Purposeful</h2>
-                    <h2>Web Development</h2>
```

Comment:
It shouldn't matter much on a project like this, but in the real world <h> tags are significant. They give a lot of information to search engines and are important for accessibility. For those considerations, imagine (or actually view) your page with no CSS files. Here you would have two lines for two titles. With the change below,
`<h2>Purposeful <div>Web Development</div></h2>`, the div wrapper will cause a line break that most viewers will see, but without styles its a single title.